src/modelmanager.cpp

```diff
@@ -225,6 +226,62 @@ Status ModelManager::loadPipelinesConfig(rapidjson::Document& configJson) {
     return ovms::StatusCode::OK;
 }
 
+Status ModelManager::loadCustomLoadersConfig(rapidjson::Document& configJson) {
+    const auto itrp = configJson.FindMember("custom_loader_config_list");
+    if (itrp == configJson.MemberEnd() || !itrp->value.IsArray()) {
+        return StatusCode::OK;
+    }
+
+    // Load Customer Loaders as per the configuration
+    SPDLOG_INFO("Using Customloaders");
+    auto& customloaders = ovms::CustomLoaders::instance();
+    for (const auto& configs : itrp->value.GetArray()) {
+        const std::string loaderName = configs["config"]["loader_name"].GetString();
```

Comment:
Did you tested what happens when you add two custom loaders with the same name? I mean have two loaders named exactly the same & added into config.json in one go? Will user get error message for that?

Reply:
At this pint second declaration in config file is ignore as we add to the new list only if loader with same name does not exist. No error is returned from config loader. Either we can live with this or we can make it similar to what is done with model specification with same name. In case of model with same name, the config is rewritten with values from second declaration. Please suggest

Reply:
In the current code second declaration is ignore by printing an error to logs

Reply:
Ok then it seems that we have a bug in OVMS if we don;t log that admin tries to load 2 models with the same name.
From my point of view it is better to load first config & then report error if there is second one in configuration file:
This way we still:
1) Inform the admin that they did something wrong
2) Don't waste time on double processing
That being said just logging the error is still ok. The rest is less important.